ruby/cross_service_examples/item-tracker/spec/db_wrapper_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "multi_json"
+require "yaml"
+require "json"
+require "rspec"
+require "aws-sdk-rdsdataservice"
+require "aws-sdk-ses"
+require_relative("../db_wrapper")
+require_relative("../report")
+
+describe "CRUD commands on Aurora" do
+  client = Aws::RDSDataService::Client.new
+  config = YAML.safe_load(File.open(File.join(File.dirname(__FILE__), "./../helpers", "config.yml")))
+  let(:wrapper) { DBWrapper.new(config, client) }
+
+  it "gets the table name" do
+    table = wrapper.get_table_name
+    expect(table).to be_an_instance_of(String)
```

Comment:
In general your tests should assert something more than the type of the return value. The return type is already specified by the function signature, so your test is really only ensuring that the function runs without errors, but you have the data (from the config in this case for table name), so you can assert against that.

Reply:
Asserting against the config data doesn't validate anything. The only thing I can validate against is the return object from the function.